hardware/esp8266com/esp8266/libraries/ESP8266WiFi/examples/NTPClient/NTPClient.ino

```diff
@@ -67,20 +67,39 @@ void setup()
   Serial.println(udp.localPort());
 }
 
+
+
 void loop()
 {
+  // this will keep track of answers we get. If we don't get answers, one reasion might be that the udp receiver 
+  // might not be working any more (for example due to interrupted and restored WIFI connection)
+  static int udperrorcounter = 0;
+
   //get a random server from the pool
-  WiFi.hostByName(ntpServerName, timeServerIP); 
+  if (!WiFi.hostByName(ntpServerName, timeServerIP)) {
+    Serial.println("Hostname lookup failed");
+    return;
+  }
 
   sendNTPpacket(timeServerIP); // send an NTP packet to a time server
   // wait to see if a reply is available
+  udperrorcounter++;
   delay(1000);
-  
+
   int cb = udp.parsePacket();
   if (!cb) {
     Serial.println("no packet yet");
+
+    if (udperrorcounter > 3) { // 3 is just randomly choosen. in case of lot packet drops...
```

Comment:
I don't get it.

Why again stopall is a bad idea, when udp has stopped working?